File: _clases/C/04control.c
#include <stdio.h>
int main(){
   float x=10;
   char signo;
   //CONDICIONALES (IF,ELSE,ELSE IF)
   if (x>0){ 
             printf( "x es positivo\n");
	     signo = "+";
   }
   else if (x<0) {
             printf( "x es negativo\n");
	     signo = "-";
   }
   else{
             printf( "x es cero\n");
	     signo = "0";
   }

   /*// SWITCHES (CASE)
   switch( signo ){
	   case '+':
         printf( "x es positivo\n");
	   case '-':
         printf( "x es negativo\n");
	   default:
         printf( "x es cero\n");
   }
   */

   //FOR
   int i=0;
   for (i=0;i<10;++i){

	printf("%3d\n",i);
   }

   //WHILE
   while (i>0){

	printf("%3d\n",i);
   	--i;
   } 
   
   return 0;
}

File: _clases/FORTRAN/04control.f90
        PROGRAM control
                implicit none
                real    :: x
                
                x=1.0
                
               !CONDICIONALES (IF,ELSE,ELSE IF)
              if (x>0) then
                        print*, "x es positivo"
               else if (x<0) then
                        print*, "x es negativo"
               else
                        print*, "x es cero"
              end if

               !SWITCHES (SELECT/CASE)
                
               select case(x)
                 case(>0)
                        print*, "x es positivo"
                 case(<0)
                        print*, 'x es negativo'
                 case default
                        print*, "x es cero"
               end select

        END PROGRAM control
        PROGRAM control
                implicit none

                integer :: i
        !DO
        do i=-6,10,2
                print*,i
        end do
    
        !DO WHILE
        i=1
        do while (i<10)
                print*, i
                i=i+1
        end do

        END PROGRAM control
